.github/workflows/ci-tests.yml

```diff
@@ -303,7 +303,9 @@ jobs:
               working-directory: /home/nominatim/nominatim-project
 
             - name: Import
-              run: nominatim import --osm-file ../test.pbf
+              run: |
+                  nominatim import --prepare-database
+                  nominatim import --no-superuser --osm-file ../test.pbf
```

Comment:
Instead of modifying the existing CI run, add another, where you set up a privileged and non-privileged user and really run the second command in non-privileged mode.

Reply:
I looked at this yesterday but I'm noticing I'm a bit out of my depth in that regard. Can you give me a pointer to where the users are created? Once I have that it should be pretty easy.